modules/javafx.media/src/main/java/com/sun/media/jfxmedia/locator/Locator.java

```diff
@@ -422,7 +422,7 @@ public void init() throws URISyntaxException, IOException, FileNotFoundException
                             InputStream stream = getInputStream(uri);
                             stream.close();
                             isConnected = true;
-                            contentType = MediaUtils.filenameToContentType(uri.getPath()); // We need to provide at least something
+                            contentType = MediaUtils.filenameToContentType(uri); // We need to provide at least something
```

Comment:
would it be possible to explain what ie being done instead?  i.e. "try to determine the content type based on extension" or something like that?

Reply:
Fixed.